internal/controller/ws/config.go

```diff
@@ -16,6 +16,7 @@ const (
 	PONG_WAIT           = "WebSocket_Pong_Wait"
 	PING_PERIOD         = "WebSocket_Ping_Period"
 	MAX_MESSAGE_SIZE    = "WebSocket_Max_Message_Size"
+	KNOWN_SECRETS       = "Known_Secrets"
```

Comment:
Can we rename this to something like SERVICE_TO_SERVICE_CREDENTIALS or something like that?  Just so its a little more clear about what it is.

Reply:
:+1: